src/kaltura-live-plugin.tsx

```diff
@@ -236,24 +248,29 @@ export class KalturaLivePlugin implements OnMediaUnload, OnPluginDestroy {
             return;
         }
 
+        if (receivedState === LiveBroadcastStates.Offline) {
+            this._manageOfflineSlate(OfflineTypes.NoLongerLive);
+            logger.info("Received isLive false after ended - show no longer live slate", {
+                method: "handleLiveStatusReceived"
+            });
+            return;
+        }
+
         if (receivedState === LiveBroadcastStates.Live) {
             if (this.reloadMedia) {
-                // if playback ended OR player got "abort" event
-                logger.info("Video ended and isLive is true. Reset player engine", {
+                logger.info("Player got error but stream is Live now. Reload media", {
                     method: "handleLiveStatusReceived"
                 });
-                this._reloadMedia();
+                this._loadMedia();
```

Comment:
why in error we not using the same flow of detach/attach?

Reply:
we can't use attach\detach media on errors because on error we destroy media source adapter, so it not available anymore